overlays/overlay-list.nix

```diff
@@ -3,4 +3,5 @@
   ./niv.nix
   ./nixops.nix
   ./nginx-monitoring.nix
+  ../globals.nix
```

Comment:
this should not end-up in the [`overlays` attribute](https://github.com/input-output-hk/ops-lib/blob/master/default.nix#L6) that is used by cardano-ops. Probably best to move it to https://github.com/input-output-hk/ops-lib/blob/28a134afad0393b1adc824efbd41c1ba9fe80011/nix/default.nix#L6